Hunting Queries/Microsoft 365 Defender/General queries/Endpoint Linux Agent Health Status Report

```diff
@@ -0,0 +1,28 @@
+
+name: Endpoint Linux Agent Health Status Report
+description: |
+    This query will provide a report on Linux configurations for Microsoft Defender Endpoint Protection.
+    Tests which are reporting "BAD" as a result imply that the associated capability is not configured per best practice recommendation.
+requiredDataConnectors:
+- connectorId: MicrosoftThreatProtection
+  dataTypes:
+  - DeviceTvmSecureConfigurationAssessment
+tactics:
```

Comment:
In the tactics section we need to have relevant MITRE Tactics. https://attack.mitre.org/tactics/enterprise/

If this is not applicable the field should be left blank. 
Something like tactics: []